static/docs/user-guide/external-data/amazon.md

```diff
@@ -0,0 +1,77 @@
+# External Data on Amazon S3
+
+Amazon S3 or Amazon Simple Storage Service is a service offered by Amazon Web
+Services (AWS) that provides object storage through a web service interface.
+
+## External Dependencies
+
+Let's take as an example a stage that simply downloads a file from an Amazon S3
+bucket:
+
+```dvc
+$ dvc run \
+      -d s3://mybucket/file.csv \
+      -o file.csv \
+      'aws s3 cp s3://mybucket/file.csv file.csv'
+```
+
+It is even easier to download a file if we use the command `dvc import-url`:
+
+```dvc
+$ dvc import-url s3://mybucket/file.csv
+```
+
+Or, using a remote:
```

Comment:
remote -> remote notation?

Reply:
or put `remote`

Reply:
it's kinda not only remote as usual it's also a protocol, so some explanation is needed

Reply:
> remote -> remote notation?

First we create a remote (with `dvc remote add`), so "remote" is ok.
I think that there is no need to introduce the term "remote notation", because it is kind of obvious.